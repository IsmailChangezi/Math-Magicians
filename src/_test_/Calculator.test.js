import React from 'react';
import renderer from 'react-test-renderer';

import Calculator from '../components/calculator';

it('matches snapshot', () => {
  const tree = renderer.create(<Calculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
