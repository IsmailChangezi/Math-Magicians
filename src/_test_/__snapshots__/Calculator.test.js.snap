// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  id="calculator"
>
  <input
    id="result"
    name="calculator-screen"
    type="text"
    value={0}
  />
  <div
    id="calcButtons"
  >
    <div>
      <button
        className="calcButton"
        id="clear"
        name="AC"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="calcButton"
        id="sign"
        name="+/-"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="calcButton"
        id="percent"
        name="%"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="calcButton operator"
        id="divide"
        name="รท"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div>
      <button
        className="calcButton"
        id="7"
        name="7"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="calcButton"
        id="8"
        name="8"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="calcButton"
        id="9"
        name="9"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="calcButton operator"
        id="multiply"
        name="x"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div>
      <button
        className="calcButton"
        id="4"
        name="4"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="calcButton"
        id="5"
        name="5"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="calcButton"
        id="6"
        name="6"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="calcButton operator"
        id="less"
        name="-"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div>
      <button
        className="calcButton"
        id="1"
        name="1"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="calcButton"
        id="2"
        name="2"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="calcButton"
        id="3"
        name="3"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="calcButton operator"
        id="plus"
        name="+"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div>
      <div>
        <button
          className="calcButton2"
          id="button17"
          name="0"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
      </div>
      <div>
        <button
          className="calcButton"
          id="button18"
          name="."
          onClick={[Function]}
          type="button"
        >
          .
        </button>
        <button
          className="calcButton operator"
          id="button19"
          name="="
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
